generated/proto/rule.proto

```diff
@@ -29,6 +29,8 @@ message MappingRuleSnapshot {
   int64 cutover_time = 3;
   map<string,string> tag_filters = 4;
   repeated Policy policies = 5;
+  int64 last_updated_at = 6;
```

Comment:
suggestion(take/leave): I like to align word boundaries (within reason) for proto definitions for readability. 

something like ~

```
message MappingRuleSnapshot {
   string             name            = 1;
   bool               tombstoned      = 2;
   int64              cutover_time    = 3;
   map<string,string> tag_filters     = 4;
   repeated Policy    policies        = 5;
   int64              last_updated_at = 6;
```

ps https://github.com/godlygeek/tabular is a godsend for this kinda thing

Reply:
Unless you're suggesting we add this thing as part of `make proto-gen` this formatting will survive exactly one version of the proto :)

Reply:
`make proto-gen` generates the `*.go` bindings corresponding to the `.proto`, it doesn't modify the `.proto` file at all.